src/main/java/com/homework/mateakademy/config/WebSecurityConfig.java

```diff
@@ -0,0 +1,38 @@
+package com.homework.mateakademy.config;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import com.homework.mateakademy.service.UserService;
+
+
+@Configuration
+@EnableWebSecurity
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+    @Autowired
```

Comment:
Field injection is not recommended 
http://blog.marcnuri.com/field-injection-is-not-recommended/